packages/members-events-service/lib/last-seen-at-updater.js

```diff
@@ -44,7 +52,8 @@ class LastSeenAtUpdater {
     async updateLastSeenAt(memberId, memberLastSeenAt, timestamp) {
         const timezone = this._settingsCacheService.get('timezone');
         if (memberLastSeenAt === null || moment(moment.utc(timestamp).tz(timezone).startOf('day')).isAfter(memberLastSeenAt)) {
-            await this._memberModel.edit({
+            const membersApi = await this._getMembersApi();
```

Comment:
👍 Thanks for following the existing pattern!

However, this pattern was used because at one point we used to reinstantiate the members service throughout the lifetime of the application - which mean you needed a function to get the active instance.

This is no longer the case and there's a single instance throughout the application lifetime!

There's no need to change this now, but just a heads up on why that exists ☺️

Reply:
Yeah, I was a bit confused about the pattern 🙈

Reply:
😁 S'all good - another one on the list to cleanup!